synapse/cortex.py

```diff
@@ -854,6 +854,14 @@ class Cortex(s_oauth.OAuthMixin, s_cell.Cell):  # type: ignore
             'description': 'Enable Storm scrape interfaces when using $lib.scrape APIs.',
             'type': 'boolean',
         },
+        'storm:query:mirror:url': {
+            'description': 'Telepath URL for a Cortex mirror or AHA pool to offload Storm queries to. This is an experimental option and should not be used in production.',
+            'type': 'string',
+        },
+        'storm:query:mirror:timeout': {
+            'description': 'Timeout to wait for a mirror to catch up before running a query on the leader.',
```

Comment:
Do we want to differentiate the timeout waiting for a pool member to be online vs waiting for nexus to catch up?